terraform/app.tf

```diff
@@ -157,14 +149,18 @@ resource "aws_ssm_parameter" "tea_mapping" {
   name = "${local.app_path}/swodlr.tea-mapping.${each.key}"
   type = "String"
   value = each.value
-  overwrite = true
 }
 
 resource "aws_ssm_parameter" "app_product_create_queue_url" {
   name = "${local.app_path}/swodlr.product-create-queue-url"
   type = "SecureString"
   value = aws_sqs_queue.product_create.url
-  overwrite = true
+}
+
+resource "aws_ssm_parameter" "available_tiles_table_name" {
+  name = "${local.app_path}/swodlr.available-tiles-table-name"
```

Comment:
do we need the `swodlr.` part after `/`?

Reply:
Yes, it's specific to namespaces in Spring. In the microservices they're not required because we're the only ones in the namespace